Components/AutoCategoryFixer.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * This class is used to revert category changes due to the option "auto-import categories" was enabled
+ *
+ * - Add missing remote categories in Connect tables
+ * - Assign remote products to these categories
+ * - Deactivate all auto imported categories
+ */
+
+namespace ShopwarePlugins\Connect\Components;
+
+
+class AutoCategoryFixer
```

Comment:
You specified the responsibility of the class in the comment header but here you use a very general name for this class. Maybe it would be better to name it `AutoCategoryReverter` or something else?